apps/dashboard/src/app/retrieval-agent/agent-dashboard/agent-dashboard.component.scss

```diff
@@ -2,6 +2,8 @@
 @use 'agent-dashboard.tokens' as dashboard;
 
 .agent-dashboard {
+  position: relative;
```

Comment:
This rule seems to make the sidebar div always visible (when scrolling to the right), event when the sidebar is closed. I don't know if it's a bug or not.

Reply:
definitely a bug yes, good catch! I added this property when I tried to add a mask when loading the workflow, but it was giving a blinking effect worst than not having a mask, so I removed the mask and forgot this property…
I'll remove it now so I don't forget, thanks!!